src/squlearn/util/executor.py

```diff
@@ -31,6 +33,15 @@
 from qiskit import QuantumCircuit
 
 
+if version.parse(qiskit.__version__) <= version.parse("0.45.0"):
```

Comment:
Yeah it is a bit tricky, since qiskit uses both and it is a bit hard to tell when they do. They cleaned this in Qiksit 1.0. But I think we should support <1.0 as well for the moment